themes-default/slim/views/vue-components/anidb-release-group-ui.mako

```diff
@@ -145,33 +160,33 @@ Vue.component('anidb-release-group-ui', {
 
             this.sendValues();
         },
-        deleteFromList: function(list) {
-            this.allReleaseGroups = this.allReleaseGroups.filter(x => {
-                if (x.memberOf !== list || !x.toggled) {
-                    return x;
-                }
-            })
+        deleteFromList(list) {
+            this.allReleaseGroups = this.allReleaseGroups.filter(x => x.memberOf !== list || !x.toggled);
 
             this.sendValues();
         }
     },
     computed: {
-        itemsWhitelist: function() {
+        itemsWhitelist() {
             return this.allReleaseGroups.filter(x => x.memberOf == 'whitelist');
         },
-        itemsBlacklist: function() {
+        itemsBlacklist() {
             return this.allReleaseGroups.filter(x => x.memberOf == 'blacklist');
         },
-        itemsReleaseGroups: function() {
+        itemsReleaseGroups() {
             return this.allReleaseGroups.filter(x => x.memberOf == 'releasegroups');
         },
-        showDeleteFromWhitelist: function() {
+        showDeleteFromWhitelist() {
             return this.allReleaseGroups.filter(x => x.memberOf == 'whitelist' && x.toggled == true).length !== 0;
         },
-        showDeleteFromBlacklist: function() {
+        showDeleteFromBlacklist() {
             return this.allReleaseGroups.filter(x => x.memberOf == 'blacklist' && x.toggled == true).length !== 0;
         }
-
+    },
+    watch: {
+        allReleaseGroups() {
```

Comment:
You have to define allReleaseGroups as a function? It will now to watch the variable?

Reply:
More or less.